### ISSUE TITLE
Dune executable should accept `--help` and `--version` arguments

### ISSUE THREAD

**Issue by Aloso** [adam-mcdaniel/dune#6]
Typing `dune --version` should print version information. `dune --help` should print help information or open the man page.

Also, many shells accept a `-c, --command` argument to execute commands and print the result, e.g.

   ```
   ~ $ bash -c "echo hello world"
   hello world
   ~ $
   ```

**Comment by Aloso** [comment]
@adam-mcdaniel  I'd like to use [`clap 3.0.0-beta.5`](https://docs.rs/clap/3.0.0-beta.5/clap/index.html) for this.

**Comment by adam-mcdaniel** [comment]
Yes, I agree. I love clap a lot: I think it would be great for this. I can add it.
